packages/analytics-types/src/element-interactions.ts

```diff
@@ -50,6 +50,14 @@ export interface ElementInteractionsOptions {
    */
   pageUrlAllowlist?: (string | RegExp)[];
 
+  /**
+   * List of page URLs to exclude from auto tracking.
+   * When provided, tracking will be blocked on these URLs.
+   * Both full URLs and regex are supported.
+   * This is checked after pageUrlAllowlist.
+   */
+  pageUrlExcludelist?: (string | RegExp)[];
```

Comment:
Remove as it analytics-types is outdated now

Reply:
done